src/app/edit.component/edit.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core'
```

Comment:
Is this file is app.component.ts or edit.component.ts because it has all same configuration.

Reply:
sir it is edit.component.ts file but that one which show on app.component file i forget to remove .

Reply:
then remove it